spec/requests/api/v3/render_resource_spec.rb

```diff
@@ -67,28 +67,45 @@
               'Hello World! This *is* textile with a ' +
                 '"link":http://community.openproject.org and ümläutß.'
             end
-            let(:text) do
-              '<p>Hello World! This <strong>is</strong> textile with a ' +
+
+            it_behaves_like 'valid response' do
+              let(:text) do
+                '<p>Hello World! This <strong>is</strong> textile with a ' +
                 '<a href="http://community.openproject.org" class="external">link</a> ' +
                 'and ümläutß.</p>'
+              end
             end
-
-            it_behaves_like 'valid response'
           end
 
           context 'with context' do
-            let(:context) { api_v3_paths.work_package work_package.id }
             let(:params) { "Hello World! Have a look at ##{work_package.id}" }
             let(:id) { work_package.id }
             let(:href) { "/work_packages/#{id}" }
             let(:title) { "#{work_package.subject} (#{work_package.status})" }
-            let(:text) {
-              "<p>Hello World! Have a look at <a href=\"#{href}\" "\
-              "class=\"issue work_package status-1 priority-1\" "\
-              "title=\"#{title}\">##{id}</a></p>"
-            }
 
-            it_behaves_like 'valid response'
+            context 'with work package context' do
+              let(:context) { api_v3_paths.work_package work_package.id }
+
+              it_behaves_like 'valid response' do
+                let(:text) {
+                  "<p>Hello World! Have a look at <a href=\"#{href}\" "\
+                  "class=\"issue work_package status-1 priority-1\" "\
+                  "title=\"#{title}\">##{id}</a></p>"
+                }
+              end
+            end
+
+            context 'with project context' do
+              let(:context) { "/api/v3/projects/#{work_package.project_id}" }
+
+              it_behaves_like 'valid response' do
+                let(:text) {
+                  "<p>Hello World! Have a look at <a href=\"#{href}\" "\
+                  "class=\"issue work_package status-1 priority-1\" "\
+                  "title=\"#{title}\">##{id}</a></p>"
```

Comment:
Am I wrong or is the expected text the same for both, the project and the work package context?

I am asking because in that case you could have left it in its old position and thus shared it among both contexts...